example/1.terraform-automation/README.md

```diff
@@ -0,0 +1,95 @@
+# Terraform Automation Source
+
+
+## Introduction
+
+This repository contains the terraform modules which helps in automating the GCP resource creation. These terraform module will create
+- Cloud Composer
+- GCS bucket
+- Cloud Build Trigger
+- Cloud Source Repository
+- Service Account for Cloud Composer
+- Pub-Sub respource
+- Network resources like VPC and Subnet
+- Enable Google API's
+
+## Pre requistie
+
+1. You must have a GCP project created with `project owner` permissions.
+2. Terraform and Gcloud cli are installed in your machine.
+
+## Execution
+
+1. git clone this repository and `cd ci-cd-for-data-processing-workflow/example/1.terraform-automation`.
+2. Perform a gcloud login using `gcloud auth application-default login`
+3. Update `terraform.tfvars` with the values.
```

Comment:
explicitly spell out what need to be updated, it looks like project id is that only thing that need to be provided, everything else have default values. if it's case, state is here